src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -78,23 +79,23 @@
         :valid-ens-or-address? valid-ens-or-address?}])))
 
 (defn- ens-linked-address
-  [{:keys [address networks theme]}]
-  [quo/text
-   {:size  :paragraph-2
-    :style {:padding-horizontal 12
-            :padding-top        4}}
-   (map (fn [network]
-          ^{:key (str network)}
-          [quo/text
-           {:size  :paragraph-2
-            :style {:color (colors/resolve-color network theme)}}
-           (str (subs (name network) 0 3) ":")])
-        networks)
-   [quo/text
-    {:size   :paragraph-2
-     :weight :monospace
-     :style  {:color (colors/theme-colors colors/neutral-100 colors/white theme)}}
-    address]])
+  [{:keys [address networks]}]
+  (let [theme (quo.theme/use-theme)]
+    [quo/text
+     {:size  :paragraph-2
+      :style {:padding-horizontal 12
```

Comment:
Multiline styles should be in style namespace.

Reply:
Done